### PR TITLE
Add stackadapt browser destination docs

### PR DESCRIPTION
**Pull request by omfgitsjack**
### Proposed changes
- Add new documentation for Stackadapt's browser destination action

### Merge timing

- ASAP once approved

### CODE REVIEW

File: src/connections/destinations/catalog/stackadapt/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+{% include content/ajs-upgrade.md %}
+
+<!-- In the section below, explain the value of this actions-based destination. If you don't have a classic version of the destination, remove this section. -->
+
+## Benefits of StackAdapt (Actions)
+
+StackAdapt (Actions) provides the following benefits:
+
+- **Event & Page View Tracking**. Leverage the StackAdapt Pixel for event and page view tracking
```
Comment (forstisabella) [inline src/connections/destinations/catalog/stackadapt/index.md:24]:
```suggestion
- **Event & Page View Tracking**: Leverage the StackAdapt Pixel for event and page view tracking
```

File: src/connections/destinations/catalog/stackadapt/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- **Audience Retargeting**. Use pixel data to create lookalike audiences
```
Comment (forstisabella) [inline src/connections/destinations/catalog/stackadapt/index.md:25]:
```suggestion
- **Audience Retargeting**: Use pixel data to create lookalike audiences
```

File: src/connections/destinations/catalog/stackadapt/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- **Reporting**. Consolidate all conversion and other media metrics into one platform, for a robust reporting solution
```
Comment (forstisabella) [inline src/connections/destinations/catalog/stackadapt/index.md:26]:
```suggestion
- **Reporting**: Consolidate all conversion and other media metrics into one platform, for a robust reporting solution
```

**Review by forstisabella — APPROVED** [review]
Looks good to go!

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**